control_panel_api/permissions.py

```diff
@@ -37,7 +42,12 @@ def has_permission(self, request, view):
             return False
 
         path = request.path.lower()
-        return path.startswith(f'/k8s/api/v1/namespaces/user-{username}')
+
+        for api in self.ALLOWED_APIS:
+            if path.startswith(f'/k8s/{api}/namespaces/user-{username}'):
+                return True
```

Comment:
I'd be tempted to change this to get the API from the path (`/(api\/.*)\/namespaces/`) and check it is in `ALLOWED_APIS` instead of looping, but this is fine as is.